templates/terraform/resource.erb

```diff
@@ -189,20 +189,28 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
     }
 <%    end -%>
 <%  end -%>
+    billingProject := ""
+
+    billingProject, _ = getBillingProject(d, config)
```

Comment:
With this check here, the billing_project will be overriden by the direct/indirect project on resources that have them. I think we want this to appear after both of those.